install_requirements.R

```diff
@@ -0,0 +1,6 @@
+#############
+# Required libraries to install
+#############
+install.packages('tidyverse')
+install.packages("synapser", repos=c("http://ran.synapse.org", "http://cran.fhcrc.org"))
+install.packages("synapserutils", repos=c("http://ran.synapse.org", "http://cran.fhcrc.org"))
```

Comment:
This package is currently installing the old pythonEmbedInR. I would install from source using the remotes R package

Reply:
Tom, the current R version offered in Linux Notebook Instance is 3.6.3. Installing synapser via remotes - `remotes::install_github('https://github.com/Sage-Bionetworks/synapser.git')`. 

I get the following error - 
```ERROR: this R is version 3.6.3, package 'synapser' requires R >= 4.0
Warning message:
In i.p(...) :
  installation of package ‘/tmp/RtmpJhh4gY/file449c4c06896f/synapser_1.0.0.tar.gz’ had non-zero exit status```
  
 So I won't change this now.

Reply:
Thats unfortunate, I know for a fact that we have been encouraging people to use the docker solution instead of the notebook solution due to the old R version.

That said, these instructions have to be updated... https://help.sc.sageit.org/sc/Service-Catalog-Provisioning.938836322.html#ServiceCatalogProvisioning-RunanRStudioNotebook